tests/baselines/reference/genericRestArityStrict.errors.txt

```diff
@@ -1,15 +1,19 @@
-tests/cases/conformance/types/rest/genericRestArityStrict.ts(7,6): error TS2345: Argument of type '(x: number, y: number) => number' is not assignable to parameter of type '() => void'.
+tests/cases/conformance/types/rest/genericRestArityStrict.ts(7,1): error TS2554: Expected 3 arguments, but got 1.
+tests/cases/conformance/types/rest/genericRestArityStrict.ts(8,1): error TS2554: Expected 3 arguments, but got 8.
 
 
-==== tests/cases/conformance/types/rest/genericRestArityStrict.ts (1 errors) ====
+==== tests/cases/conformance/types/rest/genericRestArityStrict.ts (2 errors) ====
     // Repro from #25559
     
     declare function call<TS extends unknown[]>(
         handler: (...args: TS) => void,
         ...args: TS): void;
       
     call((x: number, y: number) => x + y);

```

Comment:
Is this improved error because of the variance inference change?

Reply:
Yes